src/components/ChartNode.js

```diff
@@ -87,19 +87,21 @@ const ChartNode = ({
 
   const addArrows = e => {
     const node = e.target.closest("li");
-    const parent = node.parentNode.closest("li");
-    const isAncestorsCollapsed =
-      node && parent
-        ? parent.firstChild.classList.contains("hidden")
-        : undefined;
-    const isSiblingsCollapsed = Array.from(
-      node.parentNode.children
-    ).some(item => item.classList.contains("hidden"));
+    if (node) {
+      const parent = node.parentNode.closest("li");
+      const isAncestorsCollapsed =
+        node && parent
```

Comment:
Can remove `node` here since checking has already been added in line 90

Reply:
Thank you for noticing it. I pushed another update re: this.